app/assets/stylesheets/application.css.scss

```diff
@@ -127,3 +89,19 @@ $image-path: '';
     display: block;
   }
 }
+
+.usa-button--tiny {
+  border-radius: .25rem;
+  font-size: .6654411765rem;
+  padding: .5rem;
+}
+
+//  TODO: this may be updated in a future Login.gov design system release.
+// remove this if/when it is.
+.usa-nav__primary button {
+  &[aria-expanded='true'] {
+    &:hover {
+      background-color: $theme-color-primary-darker;
+    }
+  }
+}
```

Comment:
Based on [the upstream fix](https://github.com/18F/identity-design-system/pull/419), I suspected this may have an inverse effect of regressing the intended mobile appearance:

![image](https://github.com/18F/identity-dashboard/assets/1779930/687ef666-f6ff-4dad-a1b5-cacb4b131dbc)

You can probably use the viewport helper mixin like in the design system pull request to fix it:

```suggestion
//  TODO: this may be updated in a future Login.gov design system release.
// remove this if/when it is.
@include at-media-max($theme-header-min-width) {
  .usa-nav__primary button {
    &[aria-expanded='true'] {
      &:hover {
        background-color: $theme-color-primary-darker;
      }
    }
  }
}
```

Reply:
fixed it to be `at-media($theme-header-min-width) {` and it works! thanks!